course_inventory/inventory.py

```diff
@@ -182,6 +182,31 @@ def gather_course_data_from_api(account_id: int, term_ids: Sequence[int]) -> pd.
     return course_df
 
 
+def gather_account_data_from_api(account_ids: Sequence[int]) -> pd.DataFrame:
+    logger.info('** gather_account_data_from_api')
```

Comment:
It would be nice to see which method is producing the log message, but this also seems like it could lead to a lot of repetitive code if you're doing it in every method. Also, I agree, it would be nice to get the name of the method dynamically rather than hardcode it as a string. It sounds to me like this is a separate issue, and not one I want to address in this PR.

Reply:
It's a single line of repeated code.  It'd be nice to have a decorator for this purpose.